pkg/cmd/completion/completion.go

```diff
@@ -72,7 +75,7 @@ func Command() *cobra.Command {
 		Annotations: map[string]string{
 			"commandType": "utility",
 		},
-		Args: cobra.MinimumNArgs(1),
+		Args: cobra.ExactArgs(1),
```

Comment:
we can use `ValidArgs []string` here, we don't need default then

Reply:
Actually I was thinking if we use `Args: cobra.ExactValidArgs(1)` then we don't require the default block in the switch case and also explicit check of the len(args) before switch.

Reply:
@rupalibehera what does the `ExactValidArgs` do ? aka how does it know which value are valid ?

Reply:
it checks if the input is present in the `ValidArgs: []string{"bash", "zsh", "fish", "powershell"}` and also check if the number of parameter is correct, so we don't have to explicitly check in the code.

Reply:
Oh nice, I like that 😉